cmd/liqoctl/cmd/peer.go

```diff
@@ -90,8 +91,11 @@ func newPeerCommand(ctx context.Context, f *factory.Factory) *cobra.Command {
 
 	// Networking flags
 	cmd.Flags().BoolVar(&options.NetworkingDisabled, "networking-disabled", false, "Disable networking between the two clusters")
+	cmd.Flags().Var(options.ServerServiceLocation, "server-service-location",
+		fmt.Sprintf("Location of the service to expose the Gateway Server (%q or %q). Default: %q",
```

Comment:
Not sure I would understand correctly.
I would say this way:
fmt.Sprintf("Cluster that acts as gateway server when setting up the network between two Liqo instances (%q or %q). Default: %q",